src/pages/library/dataBrowser-utils.ts

```diff
@@ -40,34 +57,37 @@ export const getDatasetReleasePoliciesDisplayInformation = dataUsePermission =>
   )
 }
 
-export const DatasetReleasePolicyDisplayInformation = ({ dataUsePermission }) => {
+export const DatasetReleasePolicyDisplayInformation = ({ 'TerraDCAT_ap:hasDataUsePermission': dataUsePermission }: { 'TerraDCAT_ap:hasDataUsePermission': string }) => {
   const { label, description } = getDatasetReleasePoliciesDisplayInformation(dataUsePermission)
   return h(Fragment, [
     label,
     description && div({ style: { fontSize: '0.625rem', lineHeight: '0.625rem' } }, [description])
   ])
 }
 
-export const isExternal = dataset => Utils.cond(
+export const isExternal = (dataset: DatasetResponse) => Utils.cond(
```

Comment:
Now that we have types here, I'd like to bring up again the idea of writing these methods as member functions on `Dataset` (or `DatasetResponse`).

So instead of writing
```
export const isExternal = (dataset: DatasetResponse) => Utils.cond(
```

We'd have 
```
export interface Dataset extends DatasetResponse {
  isExternal() {
    Utils.cond(...) as boolean
 }
}
```

These seem equivalent to me and would avoid the need to import every method that acts upon a dataset.

Reply:
If you're on the fence about this, don't do anything now and I'll PR it separately so we can discuss this change by itself.

Reply:
I can poke at that. Let me try and see.

Reply:
Discussed offline! Will keep in mind for a future improvement.